did-pkh/did-pkh-method-draft.md

```diff
@@ -104,26 +103,23 @@ generating.
 
 ### Networks
 
-*Note: Unlike
-[DID:Ethr](https://github.com/decentralized-identity/ethr-did-resolver/blob/master/doc/did-method-spec.md#method-specific-identifier)
-which defaults to mainnet but can accept an additional prefix for other
-networks, or [DID:BTCR](https://w3c-ccg.github.io/didm-btcr/#txref) which does
-not specify a network and leaves network selection/discovery to implementations,
-did:pkh currently defaults to mainnet (or equivalent) on every supported
-blockchain without a mechanism to override and select alternatives.*
-
+Note that networks (i.e., EVMs) and specific chains (i.e., ledgers, including
+private DLTs and test-nets) have to be specified separately and explicitly for
+all did-pkh addresses; in blockchain systems where accounts are controlled by
+multiple keytypes, like Tezos, the network and chain subdomains will not be
+enough to identify keytype, which must be detected from the address itself.
 
-|network id|CAIP-2 chain id|verification method type|URL for context definition|
+|account type|network id (CAIP-2) + chain id (CAIP-10)|verification method type|URL for context definition|
 |---|---|---|---|
-|`tz` (tz1)|`tezos:NetXdQprcVkpaWU`|Ed25519PublicKeyBLAKE2BDigestSize20Base58CheckEncoded2021|https://w3id.org/security#Ed25519PublicKeyBLAKE2BDigestSize20Base58CheckEncoded2021|
-|`tz` (tz2)|`tezos:NetXdQprcVkpaWU`|EcdsaSecp256k1RecoveryMethod2020|https://identity.foundation/EcdsaSecp256k1RecoverySignature2020#EcdsaSecp256k1RecoveryMethod2020|
-|`tz` (tz3)|`tezos:NetXdQprcVkpaWU`|P256PublicKeyBLAKE2BDigestSize20Base58CheckEncoded2021|https://w3id.org/security#P256PublicKeyBLAKE2BDigestSize20Base58CheckEncoded2021|
-|`eth`|`eip155:1`|EcdsaSecp256k1RecoveryMethod2020|https://identity.foundation/EcdsaSecp256k1RecoverySignature2020#EcdsaSecp256k1RecoveryMethod2020|
-|`celo`|`eip155:42220`|EcdsaSecp256k1RecoveryMethod2020|https://identity.foundation/EcdsaSecp256k1RecoverySignature2020#EcdsaSecp256k1RecoveryMethod2020|
-|`poly`|`eip155:137`|EcdsaSecp256k1RecoveryMethod2020|https://identity.foundation/EcdsaSecp256k1RecoverySignature2020#EcdsaSecp256k1RecoveryMethod2020|
-|`sol`|`solana`|Ed25519VerificationKey2018|https://w3id.org/security#Ed25519VerificationKey2018|
```

Comment:
Should this say
`solana:4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ`
for consistency? 🔍

(and good catch, btw, I didn't think to scour the CAIP repo for the mainnet chain_id for solana)

Reply:
Yes! Fixed: https://github.com/spruceid/ssi/compare/110732299262a09f614b1fcf349ffb6f89012ee8..85aab522d53d22fdfeab4473cd0ce418730e9b03